mitohifi/2.2/Dockerfile

```diff
@@ -0,0 +1,70 @@
+# Base Image
+FROM ubuntu:18.04
+
+# Metadata
+LABEL base_image="ubuntu:18.04"
+LABEL version="2.2.1"
```

Comment:
Version is dockerfile version, 1 in this case

Software.version is upstream version